simbev/helpers/helpers.py

```diff
@@ -230,3 +226,10 @@ def timeit_wrapper(*args, **kwargs):
             return result
         return timeit_wrapper
     return decorator
+
+
+def interpolate_charging_curve(x, y):
+
+    f = interp1d(x, y, kind='cubic', fill_value='extrapolate')
+
+    return f
```

Comment:
Could we plot the full charging curves to show how they look => tells the user what we do with the input data

Reply:
For now we will just show the plots for out standard profiles in the documentation